lib/index.ts

```diff
@@ -7,13 +7,27 @@ declare const cy: any
 Cypress.Commands.add('percySnapshot', (name: string, options: any = {}) => {
   const percyAgentClient = new PercyAgent({
     clientInfo: clientInfo(),
-    environmentInfo: environmentInfo()
+    environmentInfo: environmentInfo(),
+    postSnapshotDirectly: false
   })
 
   name = name || cy.state('runnable').fullTitle()
 
   cy.document().then((doc: Document) => {
     options.document = doc
-    percyAgentClient.snapshot(name, options)
+    const domSnapshot = percyAgentClient.snapshot(name, options)
+    return cy.request({
+      method: 'POST',
+      url: `http://localhost:${percyAgentClient.port}/percy/snapshot`,
+      body: {
+        name,
+        url: doc.URL,
+        enableJavaScript: options.enableJavaScript,
+        widths: options.widths,
+        clientInfo,
+        environmentInfo,
+        domSnapshot
+      }
+    })
```

Comment:
@djones we probably don't want to fail the tests if this HTTP request fails, or do we?

We can set `failOnStatusCode: false` here so that if we get a 404 or a 500 or something else that is not a 2xx or 3xx the test will continue running (cf. https://docs.cypress.io/api/commands/request.html#Syntax).

Reply:
if Percy Agent isn't available we want to gracefully allow the tests to continue.